scripts/whitesource/WhiteSource-Scan.ps1

```diff
@@ -9,9 +9,25 @@ if (-Not [System.IO.Directory]::Exists($toolsPath)){
   New-Item -Path "C:\" -Name "tools" -ItemType "directory"
 }
 
-Write-host "Download WhiteSource tool"
-$whiteSourceAgentPath = "$toolsPath\wss-unified-agent.jar"
-Invoke-WebRequest -Uri https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar -OutFile $whiteSourceAgentPath
+for ($num = 1 ; $num -le 5 ; $num++)
+{
+  try
+  {
+    Write-host "Download WhiteSource tool, attempt $num/5"
+    $whiteSourceAgentPath = "$toolsPath\wss-unified-agent.jar"
+    Invoke-WebRequest -Uri https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar -OutFile $whiteSourceAgentPath
+    break
+  }
+  catch
```

Comment:
can you catch the exception and print it? because it may fail due to different reasons (unexpected), and it would be nice to see that